### PR TITLE
Database basic implementation

### PR DESCRIPTION
**Pull request by IdoElk**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from lmsweb.models import Course, User
+
+
+class TestCourse:
+    def test_add_user(self, user: User, course: Course):
+        course.add_user(user)
+        assert user.course == course  # NOQA: S101
```
Comment (yammesicka) [inline tests/test_models.py]: Maybe we should configure it in a file level configuration for flake8

File: app.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+APP_CONFIG = {
+    'host': '0.0.0.0',  # NOQA
+    'port': 80,
+    'debug': app.debug,
+    'threaded': True,
+}
+
+if __name__ == '__main__':
+    is_prod = os.getenv('env', '').lower() == 'production'
```
[the remark this answers is not in the document]
Comment (IdoElk) [reply app.py]: Since we use `.lower` it won't work without defaulting to an empty string

File: tests/test_models.py (hunk 1 above)
Comment (IdoElk) [reply tests/test_models.py]: Done

File: app.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (yammesicka) [reply app.py]: Nice catch!

**Pull request merged**